vm-wasmi/src/validation.rs

```diff
@@ -54,14 +53,16 @@ impl<'a> CodeValidation<'a> {
     /// Middleware function for `wasmi_validation::validate_module`
     ///
     /// * `input`: Custom input to validator
-    pub fn validate_module<V: Validator>(
-        self,
-        input: <V as Validator>::Input,
-    ) -> Result<Self, ValidationError> {
-        wasmi_validation::validate_module::<V>(self.0, input)
-            .map_err(ValidationError::Validation)?;
-        Ok(self)
-    }
+    // pub fn validate_module<V: Validator>(
```

Comment:
why it was commented out?

Reply:
At the moment the code used for validation is unmaintained and outdated, and does not support current wasm instruction set (especially sign_ext)

When the PR is out of draft mode i will remove all commented out code and replace with proper validation from more up-to-date libraries.

Reply:
please add this as separate ISSUE and link from commented out code.